modules/edu.gemini.giapi.client/src/main/scala/edu/gemini/giapi/client/gpi/GPIClient.scala

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package edu.gemini.giapi.client.gpi
+
+import cats.Monad
+import cats.effect.IO
+import edu.gemini.giapi.client.Giapi
+import fs2.Stream
+
+/**
+  * Client for GPI
+  */
+class GPIClient[F[_]: Monad](giapi: Giapi[F]) {
```

Comment:
I would move the `Monad` constraint to the individual methods that require it. I don't think any need it so far.

Reply:
👍